tests/unit/document/test_database_subindices.py

```diff
@@ -0,0 +1,33 @@
+from docarray import Document, DocumentArray, dataclass
+from docarray.typing import Image, Text
+
+@dataclass
+class MyDocument:
+    image: Image
+    paragraph: Text
+
+def test_database_subindices():
+    # extend with Documents, including embeddings
+    _docs = [(MyDocument( image='https://docarray.jina.ai/_images/apple.png', paragraph='hello world'))]
+
+    da = DocumentArray(
+        storage='sqlite',  # use SQLite as vector database
+        config={'connection': 'jina4.db', 'table_name': 'test4'},
```

Comment:
can u keep the `config` as a variable so that is easily seen that they are the same